src/components/atoms/AnywayLogo.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FunctionComponent, useMemo } from "react";
+import AnywayImage from "../../assets/anyway.png";
+
+interface AnywayLogoProps {
+  height: string;
+}
+
+export const AnywayLogo: FunctionComponent<AnywayLogoProps> = ({ height }) => {
```

Comment:
Why allow height only and not just give an option for full-fledged custom styling by passing a styles object?